frontend/pages/ManageControlsPage/OSSettings/cards/CustomSettings/components/ProfileUploader/components/AddProfileCard/AddProfileCard.tsx

```diff
@@ -16,11 +16,12 @@ const AddProfileCard = ({ setShowModal }: IAddProfileCardProps) => (
     <div className={`${baseClass}__card--content-wrap`}>
       <ProfileGraphic
         baseClass={baseClass}
+        title="Update configuration profile"
         message={
           <>
-            <b>Upload configuration profile</b>
+            .mobileconfig and .json for macOS, iOS, and iPadOS.
             <br />
-            For Apple (macOS, iOS, iPadOS), Windows, or Android hosts.
```

Comment:
What about Android?

Reply:
@rachaelshaw? This is the updated text from the Figma.